cmake/Modules/VolkConfigVersion.cmake.in

```diff
@@ -21,14 +21,13 @@ set(MAJOR_VERSION @VERSION_INFO_MAJOR_VERSION@)
 set(MINOR_VERSION @VERSION_INFO_MINOR_VERSION@)
 set(MAINT_VERSION @VERSION_INFO_MAINT_VERSION@)
 
-set(PACKAGE_VERSION
-  ${MAJOR_VERSION}.${MINOR_VERSION}.${MAINT_VERSION})
+set(PACKAGE_VERSION ${MAJOR_VERSION}.${MINOR_VERSION}.${MAINT_VERSION})
 
 if(${PACKAGE_FIND_VERSION_MAJOR} EQUAL ${MAJOR_VERSION})
   if(${PACKAGE_FIND_VERSION_MINOR} EQUAL ${MINOR_VERSION})
     if(NOT ${PACKAGE_FIND_VERSION_PATCH} GREATER ${MAINT_VERSION})
-      set(PACKAGE_VERSION_EXACT 1)    # exact match for API version
+      set(PACKAGE_VERSION_EXACT 1)       # exact match for API version
```

Comment:
Should this be set to 2? And where is this variable used? We set `PACKAGE_VERSION` right above this if statement. Where do we need `PACKAGE_VERSION_EXACT`?

Reply:
`PACKAGE_VERSION_EXACT` is a return variable from CMake calling this script. It should really be set to TRUE. Ditto for `PACKAGE_VERSION_COMPATIBLE`. Inside CMake, `TRUE == 1`, so either will do. My vote is to leave this as is; just simpler.

Reply:
Technically, I think in CMake so long as `PACKAGE_VERSION_EXACT` != 0, then it will be interpreted as `TRUE` ... but that's diverging from the main query ...